nidmviewerfsl/pageStyling.py

```diff
@@ -0,0 +1,69 @@
+#======================================================================
+# This file contains a collection of small functions for reading in and
+# encoding images as well as the CSS styling for FSL output pages. It
+# assumes FSL is installed.
```

Comment:
@TomMaullin: now that we actually depend on FSL, could you update the README to state this explicitly?